cluster/ci/monitoring/mixins/grafana_dashboards/ghproxy.jsonnet

```diff
@@ -15,12 +15,30 @@ local dashboardConfig = {
         uid: 'd72fe8d0400b2912e319b1e95d0ab1b3',
     };
 
+local histogramQuantileTarget(phi) = prometheus.target(
+        std.format('histogram_quantile(%s, sum(rate(github_request_duration_bucket{job="ghproxy", token_hash=~"${token}", path=~"${path}", status=~"${status}"}[5m])) by (le))', phi),
```

Comment:
I don't think we need to regex match on the token_hash, path or token here

Reply:
We need it for the `All` value, that is `.*`.
It would be nice to have a `if-else` but I do not know how to do it.

Reply:
ah